### PR TITLE
add operating system model

### PR DESCRIPTION
**Pull request by khomesh24**
add operating system model
no form is added
To add new os add via admin table

### CODE REVIEW

File: main_app/models.py [hunk 1]
```diff
@@ -36,3 +36,9 @@ class create_host_model(models.Model):
     def __str__(self):
         return self.vm_name
 
+class operating_system_model(models.Model):
+    os_name = models.CharField(max_length=15)
+    os_location = models.CharField(max_length=100)
```
Comment (yadneshk) [inline main_app/models.py]: Here you need to add another parameter that is choices=[list of all operating systems]

File: main_app/admin.py [hunk 2]
```diff
@@ -1,7 +1,8 @@
 from django.contrib import admin
-from .models import compute_resource_model,profile_model,create_host_model
+from .models import compute_resource_model,profile_model,create_host_model,operating_system_model
```
Comment (ntkathole) [inline main_app/admin.py]: Put this in ascending order and follow character rule per line (ex 80 char/lines ) 
```
from .models import (
    create_host_model,
    compute_resource_model,
    operating_system_model,
    profile_model
)
```

Comment (yadneshk) [reply main_app/admin.py]: ok we will make those change

File: main_app/admin.py [hunk 3]
```diff
@@ -1,8 +1,11 @@
 from django.contrib import admin
-from .models import compute_resource_model,profile_model,create_host_model,operating_system_model
+from .models import (compute_resource_model,
+                     create_host_model,
```
Comment (ntkathole) [inline main_app/admin.py]: Wrong indentation. It is bad practise to push new commit for each change( amend the commit )
`git push origin testing --amend`

File: main_app/models.py (hunk 1 above)
Comment (khomesh24) [reply main_app/models.py]: @yadneshk 
I think we need to add it in create_host_model class in model.py file

File: main_app/admin.py (hunk 2 above)
Comment (khomesh24) [reply main_app/admin.py]: @ntkathole 
Changes done

File: main_app/migrations/0012_auto_20171226_0419.py [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('main_app', '0011_auto_20171225_1553'),
+    ]
+
```
Comment (ntkathole) [inline main_app/migrations/0012_auto_20171226_0419.py]: nitpick, blank line

File: main_app/models.py [hunk 5]
```diff
@@ -36,3 +36,9 @@ class create_host_model(models.Model):
     def __str__(self):
         return self.vm_name
 
+class operating_system_model(models.Model):
```
Comment (ntkathole) [inline main_app/models.py]: If it is class, if letter in caps and two blank line before class according to PIP.

File: main_app/admin.py (hunk 3 above)
Comment (yadneshk) [reply main_app/admin.py]: ok we will add all changes in a single commit

File: main_app/migrations/0012_auto_20171226_0419.py (hunk 4 above)
Comment (yadneshk) [reply main_app/migrations/0012_auto_20171226_0419.py]: what do you mean?we haven't made changes in this file

File: main_app/models.py (hunk 1 above)
Comment (CURIOUSER005) [reply main_app/models.py]: Only make sure values don't get repeated

File: main_app/migrations/0012_auto_20171226_0419.py (hunk 4 above)
Comment (ntkathole) [reply main_app/migrations/0012_auto_20171226_0419.py]: @yadneshk I cant get you. I can see this file in commit. Is this auto generated file?

**Pull request merged**